apps/examples/testcase/ta_tc/audio/itc/itc_audio_main.c

```diff
@@ -74,244 +63,708 @@ static void clean_all_data(int fd, char *buffer)
 }
 
 /**
-* @testcase         audio_pcm_open_close_tc_p
+* @testcase         itc_audio_pcm_open_close_p
 * @brief            open and initlaize pcm structure
 * @scenario         Open pcm and check its validation
 * @apicovered       pcm_open, pcm_close
 * @precondition     NA
 * @postcondition    NA
 */
-static void itc_audio_pcm_open_close_tc_p(void)
+static void itc_audio_pcm_open_close_p(void)
 {
 	g_pcm = pcm_open(0, 0, PCM_IN, NULL);
-	TC_ASSERT_GT("pcm_open", pcm_get_file_descriptor(g_pcm), 0);
+	TC_ASSERT_GT_CLEANUP("pcm_open", pcm_get_file_descriptor(g_pcm), 0, pcm_close(g_pcm));
+
 	TC_ASSERT_EQ("pcm_close", pcm_close(g_pcm), 0);
 	TC_SUCCESS_RESULT();
 }
 
 /**
-* @testcase         audio_pcm_is_ready_p
+* @testcase         itc_audio_pcm_open_p_with_different_flag
+* @brief            open and initlaize pcm structure
+* @scenario         Open pcm and check its validation
+* @apicovered       pcm_open, pcm_close
+* @precondition     NA
+* @postcondition    NA
+*/
+static void itc_audio_pcm_open_p_with_different_flag(void)
+{
+	int index = 0;
+	int count = 0;
+	int ret = 0;
+	bool check = true;
+	unsigned int flag_arr[] = { PCM_IN, PCM_OUT, PCM_MMAP, PCM_NOIRQ, PCM_MONOTONIC };
```

Comment:
We do not plan to support timestamps in audio buffers hence PCM_MONOTONIC is not needed. Could you remove it? It would help to fix build error .

Reply:
okay I removed it.